ui/src/views/Settings.vue

```diff
@@ -1,63 +1,142 @@
-    <!-- TODO configuration fields -->
-    <!-- <div v-if="error.configureModule" class="bx--row">
-              <div class="bx--col">
+      </cv-column>
+    </cv-row>
+    <cv-row>
+      <cv-column>
+        <cv-tile light>
+          <cv-form @submit.prevent="configureModule">
+            <cv-text-input
+              :label="$t('settings.nethvoice_host')"
+              v-model="form.nethvoice_host"
+              placeholder="voice.example.com"
+              :disabled="loadingState"
+              :invalid-message="error.nethvoice_host"
+              ref="nethvoice_host"
+            />
+            <cv-text-input
+              :label="$t('settings.nethcti_ui_host')"
+              v-model="form.nethcti_ui_host"
+              placeholder="cti.example.com"
+              :disabled="loadingState"
+              :invalid-message="error.nethcti_ui_host"
+              ref="nethcti_ui_host"
+            />
+            <cv-toggle
+              :label="$t('settings.lets_encrypt')"
+              value="lets_encrypt"
+              :disabled="loadingState"
+              v-model="form.lets_encrypt"
+            />
+            <label class="bx--label">
+              {{ $t("settings.user_domain_selector") }}
+            </label>
+            <cv-radio-group>
+              <cv-radio-button
+                value="none"
+                name="user_domain_radio"
+                label="None"
+                :disabled="loadingState"
+                v-model="form.user_domain_select"
+              />
+              <cv-radio-button
+                value="cluster-provided"
+                name="user_domain_radio"
+                label="Provided by Cluster"
+                :disabled="loadingState"
+                v-model="form.user_domain_select"
+              />
+              <cv-radio-button
+                value="external"
+                name="user_domain_radio"
+                label="External"
+                :disabled="loadingState"
+                v-model="form.user_domain_select"
+              />
+            </cv-radio-group>
+            <cv-combo-box
+              v-if="form.user_domain_select == 'cluster-provided'"
+              :title="$t('settings.user_domain')"
+              :options="domainList"
+              :value="form.user_domain"
+              :auto-highlight="true"
+              :label="$t('settings.user_domain_placeholder')"
+              :disabled="loadingState"
+              :invalid-message="error.user_domain"
+              v-model="form.user_domain"
+              ref="user_domain"
+            />
+            <cv-text-input
+              v-if="form.user_domain_select == 'external'"
+              :label="$t('settings.user_domain')"
+              v-model="form.user_domain"
+              placeholder="domain.example.com"
+              :disabled="loadingState"
+              :invalid-message="error.user_domain"
+              ref="user_domain"
+            />
```

Comment:
What is this combox-box for?
A user domain should always be accessed using ldapproxy (local nginx instance).
For all modules, the user domain is always provided by the cluster in clear-text.

If you want to connect to an external provider, you should configured an external user domain.